### PR TITLE
chore: release notes framework

### PR DESCRIPTION
**Pull request by sfoster1**
Add some basic content to internal release notes

### CODE REVIEW

File: app-shell/build/release-notes-internal.md [hunk 1]
```diff
@@ -3,10 +3,25 @@ For more details about this release, please see the full [technical changelog][]
...
+This is still pretty early in the process, so some things are known not to work, and are listed below. Some things that may surprise you do work, and are also listed below. There may also be some littler things of note, and those are at the bottom.
+
+## Big Things That Don't Work Yet So Don't Report Bugs About Them
+- Attach, detach, and calibration flows for anything from the protocol setup page
+- Attach, detach, and calibration flows for the gripper
+- USB connectivity
+- Pipette/gripper firmware update on attach: if you need to attach a new instrument, attach it and then power-cycle the robot
+
+## Big Things That Do Work Please Do Report Bugs About Them
+- Connecting to Flexs
```
Comment (ecormany) [inline app-shell/build/release-notes-internal.md]: 🧐 `Flexes` if we must and i hope we mustn't

Comment (SyntaxColoring) [reply app-shell/build/release-notes-internal.md]: Can we do Flexen? (kidding, unless...?)

Comment (ecormany) [reply app-shell/build/release-notes-internal.md]: 3rd declension 🙃 
```
flex     flices
flicis   flicum
flici    flicibus
flicem   flices
flice    flicibus
```

Comment (sfoster1) [reply app-shell/build/release-notes-internal.md]: `connectendum flices` I believe with the gerund form

**Review by ecormany — APPROVED** [review]
Looks fine for internal and anyone else curious enough to find them on GitHub.

**Review by vegano1 — APPROVED** [review]

**Comment by vegano1** [comment]
lgtm

**Pull request merged**